pkg/gardenlet/controller/shoot/shoot/reconciler_reconcile.go

```diff
@@ -314,7 +315,12 @@ func (r *Reconciler) runReconcileShootFlow(ctx context.Context, o *operation.Ope
 		_ = g.Add(flow.Task{
 			Name: "Renewing shoot access secrets after creation of new ServiceAccount signing key",
 			Fn: flow.TaskFn(func(ctx context.Context) error {
-				return tokenrequest.RenewAccessSecrets(ctx, o.SeedClientSet.Client(), o.Shoot.SeedNamespace)
+				return tokenrequest.RenewAccessSecrets(ctx, o.SeedClientSet.Client(),
+					client.InNamespace(o.Shoot.SeedNamespace),
+					// TODO(rfranzke): Uncomment the next line after v1.85 has been released
+					//  (together with restricting the garden's/shoot's tokenrequestor to the shoot class).
+					// client.MatchingLabels{resourcesv1alpha1.ResourceManagerClass: resourcesv1alpha1.ResourceManagerClassShoot},
```

Comment:
@rfranzke this should be done together with https://github.com/gardener/gardener/blob/17ff592e734131ef746560641bdcdec3bcfce0f1/pkg/resourcemanager/controller/add.go#L126-L127